sdk/storage_blobs/src/blob/operations/acquire_lease.rs

```diff
@@ -44,13 +37,15 @@ impl AcquireLeaseBuilder {
             url.query_pairs_mut().append_pair("comp", "lease");
             self.timeout.append_to_url_query(&mut url);
 
+            let mut headers = Headers::new();
+            headers.insert(LEASE_ACTION, "acquire");
```

Comment:
It would be nice if we could get `headers.add(azure_core::LeaseAction::Acquire)` to work, but we can leave that for a future PR.